third_party/move/move-vm/runtime/src/loader/modules.rs

```diff
@@ -189,9 +197,6 @@ impl ModuleStorageAdapter {
 pub struct Module {
     id: ModuleId,
 
-    // size in bytes
-    pub(crate) size: usize,
```

Comment:
I had a different comment about (module, size) but did not realize you removed it from Module at that point - why? it seems like useful information at hand when one has the module? or can it be inferred?

Reply:
Returned